apps/condo/domains/ticket/schema/CreateResidentTicketService.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Generated by `createservice ticket.CreateResidentTicketService --type mutations`
+ */
+const { Contact } = require('@condo/domains/contact/utils/serverSchema')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+const { Ticket } = require('../utils/serverSchema')
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/ticket/access/CreateResidentTicketService')
+const { NOT_FOUND_ERROR } = require('@condo/domains/common/constants/errors')
+const { getSectionAndFloorByUnitName } = require('@condo/domains/ticket/utils/unit')
+
+const TICKET_MOBILE_SOURCE_ID = '3068d49a-a45c-4c3a-a02d-ea1a53e1febb'
```

Comment:
It should be as an argument. It's should not be hardcoded

Reply:
ok